themes/mms-onprem/page.html

```diff
@@ -50,6 +50,22 @@ <h4 class="alert-heading" style="margin-bottom: 10px;">The MongoDB Agent has bee
     </div>
   {%- endif %}
 
+  {%- if theme_backup_monitoring_eol %}
+    <div class="alert alert-info alert-dismissible fade in" role="alert" style="margin-left: 20px">
+      <button type="button" class="close" data-dismiss="alert" aria-label="Close">
+        <span aria-hidden="true">&times;</span>
+      </button>
+      <span class="alert-message">
+        <h4 class="alert-heading" style="margin-bottom: 10px;">Ops 
+          Manager will no longer support Backup and Monitoring for 
```

Comment:
Q: Is September 30 the release date for the version(s) of Ops Manager that drop this support? It's not clear to me if this change will affect all current OM versions, only 6.0, or only ones released after Sep 30.

Reply:
Asked in the [DOCSP ticket](https://jira.mongodb.org/browse/DOCSP-21728). Will await response before backporting.